common/api/core-bentley.api.md

```diff
@@ -264,6 +262,24 @@ export namespace CompressedId64Set {
 // @public (undocumented)
 export type ComputePriorityFunction<T> = (value: T) => number;
 
+// @alpha
+export enum ConcreteEntityTypes {
```

Comment:
Why are types related to iModel types (models, elements, aspects, relationships) in core-bentley instead of core-common?

Reply:
I will move this stuff to core-common.

Reply:
done